docs/sgp40_voc_index/arduino.rst

```diff
@@ -0,0 +1,110 @@
+SGP40 on Arduino Platforms
+==========================
+
+This guide describes how to get the new sgp40 working on arduino
+platforms. It offers a step by step guide to get basic funcionality working.
+
+Preperation
+-----------
+
+First of all note that the most commenly used Arduino boards have a logic level of 5v but the
+SGP40 can only run on 3.3v and lower. We offer two sugsestions how to solve this. First you could
+use a level shifter board to connect your SGP40, alternitivly you could use one of the few Arduino 
+boards which run on 3.3v. For this guide we did the later and used the Arduino Nano 33 BLE.
```

Comment:
```suggestion
boards which run at 3.3v. For this guide we did the later and used the Arduino Nano 33 BLE.
```

In general I'd use less prose to have more "meat".. maybe

"The SGP40 only runs at Voltages between 1.6V - 3.6V. Solutions include
* Using a level shifter
* Using a board like the Arduino Nano 33 BLE that runs at 3.3V"